sync-core/src/test/java/com/cloudant/sync/replication/BasicPushStrategyTest.java

```diff
@@ -185,9 +180,25 @@ private void oneDocCreatedThenUpdatedThenDeletedAndThenPush() throws Exception {
     }
 
     private void push(int expectedDocs) throws Exception {
+        this.push(expectedDocs,
+                BasicPushStrategy.DEFAULT_CHANGES_LIMIT_PER_BATCH,
+                BasicPushStrategy.DEFAULT_MAX_BATCH_COUNTER_PER_RUN,
+                BasicPushStrategy.DEFAULT_BULK_INSERT_SIZE,
+                BasicPushStrategy.DEFAULT_PUSH_ATTACHMENTS_INLINE);
+    }
+
+    private void push( int expectedDocs,
```

Comment:
It looks like you can remove this multiple-args push() because only the defaults ever get passed through (and it's confusing to have to push() methods).